tests/utils.go

```diff
@@ -252,8 +269,8 @@ func (w *ObjectEventWatcher) Timeout(duration time.Duration) *ObjectEventWatcher
 	return w
 }
 
-func (w *ObjectEventWatcher) FailOnWarnings() *ObjectEventWatcher {
-	w.failOnWarnings = true
+func (w *ObjectEventWatcher) SetWarningsPolicy(wp WarningsPolicy) *ObjectEventWatcher {
+	w.warningPolicy = wp
```

Comment:
@oshoval the `WarningsPolicy` consists of a `[]string` slice, is it enough to do a shallow copy here?

Reply:
yep, tested that the data is valid and exists after copying.